recipes/vtk/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [ `uname` == Linux ]; then
+    CC=${PREFIX}/bin/gcc
+    CXX=${PREFIX}/bin/g++
```

Comment:
We had issues earlier where linking errors would occur due to the system compiler producing builds incompatible with Anaconda's python.exe. Why would this not be an issue on conda-forge?